### PR TITLE
Clean validation when updating

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: src/form/Attributes/common/BaseInput/Form/index.js [hunk 1]
```diff
@@ -37,21 +55,31 @@ const Form = ({nodeDef, keyboardType = 'default'}) => {
...
+        node,
+        value: _value,
+        callback: false,
+        shouldJump: false,
+      });
+    },
+    [nodeDef, node, handleUpdateNode],
+  );
+
+  const deboundedUpdate = useCallback(
```
Comment (SteRiccio) [inline src/form/Attributes/common/BaseInput/Form/index.js]: it's "debouncedUpdate" maybe, not debounded...

File: src/screens/Form/components/EntityPage/components/common/EntityPanel/component.js [hunk 2]
```diff
@@ -18,13 +19,15 @@ import TableOption from './TableOption';
...
 const EntityPanel = () => {
+  const nodeDef = useSelector(formSelectors.getParentEntityNodeDef);
   const styles = useThemedStyles(_styles);
   const isEntitySelectorOpened = useSelector(
     formSelectors.isEntitySelectorOpened,
   );
 
   const isTable = useIsTable();
   const _isTablet = isTablet();
+  const isMultiple = useMemo(() => NodeDefs.isMultiple(nodeDef), [nodeDef]);
```
Comment (SteRiccio) [inline src/screens/Form/components/EntityPage/components/common/EntityPanel/component.js:30]: the nodeDef object never changes during data entry... this useMemo could be removed

**Review by SteRiccio — APPROVED** [review]

File: src/screens/Form/components/EntityPage/components/common/EntityPanel/component.js (hunk 2 above)
Comment (ramirobg94) [reply src/screens/Form/components/EntityPage/components/common/EntityPanel/component.js:30]: This is into the panel that changes when the entity selected changes,

File: src/form/Attributes/common/BaseInput/Form/index.js (hunk 1 above)
Comment (ramirobg94) [reply src/form/Attributes/common/BaseInput/Form/index.js]: true

Comment (ramirobg94) [inline src/form/Attributes/common/BaseInput/Form/index.js]:
```suggestion
  const debouncedUpdate = useCallback(
```

File: src/form/Attributes/common/BaseInput/Form/index.js [hunk 3]
```diff
@@ -67,7 +95,7 @@ const Form = ({nodeDef, keyboardType = 'default'}) => {
             NodeDefs.getTextTransform(nodeDef)
           ] || undefined
         }
-        onChangeText={setValue}
+        onChangeText={deboundedUpdate}
```
Comment (ramirobg94) [inline src/form/Attributes/common/BaseInput/Form/index.js]:
```suggestion
        onChangeText={debouncedUpdate}
```

**Pull request merged**